ez_setup/__init__.py

```diff
@@ -13,7 +13,7 @@
 
 This file can also be run as a script to install or upgrade setuptools.
 """
-import sys
+
 DEFAULT_VERSION = "0.6c7"
 DEFAULT_URL = ("http://pypi.python.org/packages/"
                "%s/s/setuptools/" % sys.version[:3])
```

Comment:
Except that it's the line 51 that should be removed, otherwise this line will fail ^ :)